app/views/admin/users/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div>
+  <h1>Hello admin</h1>
+  <h2>Displaying Square63 clients</h2>
+  <table class="table table-dark">
+    <thead>
+      <tr>
+        <th >#</th>
+        <th > Name </th>
+        <th >Role</th>
+        <th >Email</th>
+        <th >Action</th>
+      </tr>
+    </thead>
+    <tbody>
+    	<% @users.each do |u| %>
+        <% if u.user_role == User::ROLES[:client] %>
```

Comment:
If you do not want to list down admins, pick only clients through a query in controller